tx_highered/templates/tx_highered/institution_detail.html

```diff
@@ -97,17 +96,17 @@
 
         <tbody>
           <tr>
-            <th>{% gloss "Applicants" %}</th>
+            <th>Applicants</th>
             {% for a in object.admission_buckets.applicants.values %}<td>{{ a|intcomma }}</td>{% endfor %}
           </tr>
 
           <tr>
-            <th>{% gloss "Admitted" %}<br><small>(% of Applicants)</small></th>
+            <th>Admitted<br><small>(% of Applicants)</small></th>
             {% for a in object.admission_buckets.admitted.values %}<td>{{ a.0|intcomma }} <small>({{ a.1 }}%)</small></td>{% endfor %}
           </tr>
 
           <tr>
-            <th>{% gloss "Enrolled" %}<br><small>(% of Admitted)</small></th>
+            <th>Enrolledbr><small>(% of Admitted)</small></th>
```

Comment:
As far as the dummy tag goes, we're removing `glossary` from texastribune and have no plans to use this going forward, I chose to remove them to clean up the markup so future users wouldn't be confused about whether it's an active feature or not.